File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/sqlserver/XmlType.kt
package dev.typr.foundations.docs.sqlserver

import dev.typr.kotlinfoundations.*
import dev.typr.kotlinfoundations.data.*
import dev.typr.kotlinfoundations.data.Xml

@Suppress("unused")
class XmlType {
    //start
    val xmlType: SqlServerType<Xml> = SqlServerTypes.xml
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/sqlserver/XmlType.scala
package dev.typr.foundations.docs.sqlserver

import dev.typr.foundations.{SqlServerType, SqlServerTypes}
import dev.typr.foundations.data.Xml

@SuppressWarnings(Array("unused"))
object XmlType:
  //start
  val xmlType: SqlServerType[Xml] = SqlServerTypes.xml
  //stop
